irssi/README.md

```diff
@@ -0,0 +1,29 @@
+Dockerfile-Fedora-irssi
+=======================
+
+Fedora Dockerfile for [irssi](https://irssi.org/).
+
+To Run the Container:
+---------------------
+
+```
+# docker run -it fedora/irssi
+```
+
+This will start the irssi client
+
+To Build:
+---------
+
+Copy the sources down and perform Build
+
+```
+# docker build -t <username>/irssi .
+```
+
+To Run:
+-------
+
+```
+# docker run -it <username>/irssi
```

Comment:
there are two docker runs here - they seem the same.  consolidate to one please? Also, do you really want to run it in interactive mode or detached?